src/rwe/GameHash_util.h

```diff
@@ -95,7 +95,7 @@ namespace rwe
     template <typename... Ts>
     GameHash computeHashOf(const std::variant<Ts...>& v)
     {
-        return GameHash(v.index())
+        return GameHash(static_cast<uint32_t>(v.index()))
```

Comment:
Rather than a cast here we should probably write a `computeHashOf` overload for uint64 and call it here instead of invoking the `GameHash` constructor directly.